README.md

```diff
@@ -16,6 +16,8 @@ It currently consists of
 # Release Notes
 BOAT is still under development and subject to change.
 ## 0.17.34
+* add new lint rule `B014` - to validate if examples contain all defined properties in schema
```

Comment:
New dev version is 0.17.35 please update

Reply:
The changes were on version 34. But none updated the readme so there was no conflict.